services/ui-src/src/constants.ts

```diff
@@ -79,3 +79,6 @@ export enum States {
 // TIMEOUT PARAMS
 export const IDLE_WINDOW = 30 * 60 * 1000; // ms
 export const PROMPT_AT = 29 * 60 * 1000; //ms
+
+// routes
+export const ROUTES = ["/mcpar", "/mlr", "/naaar"];
```

Comment:
nit: i'd give this a more descriptive name like `DASHBOARD_ROUTES` or something

Reply:
that's fair